services/graphql-server/src/graphql/definitions/email/campaign.js

```diff
@@ -1,11 +1,80 @@
 const gql = require('graphql-tag');
 
 module.exports = gql`
+  extend type Query {
+    emailCampaign(input: EmailCampaignQueryInput!): EmailCampaign
+      @findOne(model: "email.Campaign", using: { id: "_id" })
+    emailCampaigns(input: EmailCampaignQueryInput!): EmailCampaignConnection
```

Comment:
To match the same naming pattern used elsewhere, `emailCampaign` should be a find one (as you have), but should only accept `id` and `status` for its query input.

The `emailCampaigns` query needs to have a distinct input from `emailCampaign`, and the `EmailCampaignConnection` it returns is required.
```suggestion
    emailCampaigns(input: EmailCampaignsQueryInput!): EmailCampaignConnection!
```